DependencyInjection/Configuration.php

```diff
@@ -30,7 +27,7 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->booleanNode('autoinclude')->defaultTrue()->end()
-                ->scalarNode('base_path')->defaultValue('bundles/norzechowiczaceeditor/ace')->end()
+                ->scalarNode('base_path')->defaultValue('vendor/ace')->end()
```

Comment:
so you are expecting people to download `js` dependencies directly into folder generated by a composer called `vendor`?